core/src/main/java/cucumber/runtime/RuntimeOptions.java

```diff
@@ -180,6 +184,10 @@ private void parse(List<String> args) {
                 parsedJunitOptions.addAll(asList(arg.substring("--junit,".length()).split(",")));
             } else if (arg.equals("--wip") || arg.equals("-w")) {
                 wip = true;
+            } else if (arg.equals("--order")) {
+            	orderTypeData.put(OrderType.TYPE_NAME, args.remove(0));
```

Comment:
The names of the CLI commands are specific to the runtime options parser. It is not necessary for the `OrderType` to know what it is named on the CLI. It is okay to inline this string here.

Reply:
Will change it.